app/src/test/java/org/greenstand/android/TreeTracker/signup/SignupViewModelTest.kt

```diff
@@ -0,0 +1,126 @@
+package org.greenstand.android.TreeTracker.signup
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.google.common.truth.Truth
+import com.google.common.truth.Truth.assertThat

```

Comment:
Sorry, I didn't mean to import assertThat. I meant to use the regular Assert class or AssertJ classes. You can see what I mean in other units tests.

Can you also please remove all unneeded changes to the gradle file?